### DISCUSSION TITLE
control.phase_crossover_frequencies

### DISCUSSION THREAD

**Discussion by alittlepinetree** [python-control/python-control#1089] [Q&A]
When I used this function to calculate the phase crossover frequencies, it didn't work.
`import control
import matplotlib.pyplot as plt

G = control.tf([200.0], [1.0, 21.0, 20.0, 0.0])
x_omega, x_gain = control.phase_crossover_frequencies(G)
print(x_omega)`

**Comment by alittlepinetree** [comment]
It got two warning.

**Reply by TheStophe** [reply]
What are the warnings/errors? Also what versions of python and packages are you using, i'm sure that would be helpful. Cheers.

**Reply by alittlepinetree** [reply]
![image](https://github.com/user-attachments/assets/5d3f8130-2e6f-4a39-989e-89964f5d8bc9)

**Reply by alittlepinetree** [reply]
Python 3.11.5

**Reply by alittlepinetree** [reply]
control                   0.10.1

**Comment by slivingston** [comment]
@alittlepinetree The values in `x_omega` and `x_gain`seem reasonable to me. Do you think the values are wrong? In particular, this system has a pole at the origin, and therefore it has a phase crossover at infinity. Numerically this involves dividing by zero, hence the warning.

There is an issue to track work about not showing the warnings in this case: https://github.com/python-control/python-control/issues/1105

**Comment by murrayrm** [comment]
As noted above, the warning messages have been fixed in #1105.  Closing this discussion, but reopen if you still have questions or think there are issues.
